FWCore/ParameterSet/src/validateTopLevelParameterSets.cc

```diff
@@ -12,20 +14,35 @@
 
 namespace edm {
 
+  // NOTE: The defaults given here are not actually used when running cmsRun
+  // Those come from hard coded values in the Python code in Config.py
+  // The defaults here are used when running the edmPluginHelp utility so
+  // it is important the defaults in both places are consistent.
+
   void fillOptionsDescription(ParameterSetDescription& description) {
     description.addUntracked<unsigned int>("numberOfThreads", s_defaultNumberOfThreads)
         ->setComment("If zero, let TBB use its default which is normally the number of CPUs on the machine");
     description.addUntracked<unsigned int>("numberOfStreams", 0)
-        ->setComment("If zero, then set the number of streams to be the same as the number of threads");
+        ->setComment(
+            "If zero, then set the number of streams to be the same as the number of "
+            "Threads (except always 1 if there is a looper)");
+    description.addUntracked<unsigned int>("numberOfConcurrentLuminosityBlocks", 0)
+        ->setComment(
+            "If zero, use Framework default (currently 2). In all cases, "
```

Comment:
How about something along
```suggestion
            "If zero, use Framework default (currently 2 when numberOfStreams >= 2, otherwise 1). In all cases, "
```
?

Reply:
I agree, this could be better. Originally, I thought the second sentence covered this case. But it would be easy to get confused.

How is this?

"If zero, use Framework default (currently 2 when the number of streams >= 2, otherwise 1)."

I changed it from "numberOfStreams" because that seems to indicate the parameter value as opposed to the actual "number of streams". If the parameter numberOfStreams is 0, the default might still be 2... Trying to find a way to word this that is correct, but not too long.

Reply:
> "If zero, use Framework default (currently 2 when the number of streams >= 2, otherwise 1)."

I agree, this is better, thanks.